### ISSUE TITLE
Kernel: Non-windowed, full screen display mode

### ISSUE THREAD

**Issue by danboid** [SerenityOS/serenity#377]
As discussed in this IRC log from earlier today, it would be great if serenity had a non-windowed, non-composited full screen display mode for running games, emulators and for (non HW accelerated) video playback etc.

```
<danboid> kling, Does serenity have a full screen video mode for games and video playback?
<kling> hmm
<kling> well, there's a way to make a window full screen
<kling> but it's not like a proper full screen mode for a game/video
<kling> it's still managed by the window server
<danboid> That'd do the trick
<kling> yeah I think it might be enough, but if it's not, we can make it possible for a program to take over the screen entirely
<kling> old school style
<danboid> Might be a performance increase to be had with that I'd hope
<kling> yeah for sure
<kling> right now there's a whole bunch of pixel shuffling
<kling> necessary for a nice desktop composited from many windows
<kling> ..but if there's only one window, we don't need all that :)
```

**Comment by danboid** [comment]
I'm looking at you DooM, Quake, advancemame and qemu!

**Comment by danboid** [comment]
Closed via d92e26d0234a0719ed76dc14325998d5ed797b38

Thanks AK!
